bin/upgrade.py

```diff
@@ -22,15 +25,37 @@ def upgrade_data(p: problem.Problem, bar: ProgressBar) -> None:
             old_path.rename(new_path)
 
 
+def upgrade_testdata_yaml(p: problem.Problem, bar: ProgressBar) -> None:
+    rename = [
+        ("output_validator_flags", "output_validator_args"),
+        ("inut_validator_flags", "inut_validator_args"),
+    ]
+
+    for f in (p.path / "data").rglob("testdata.yaml"):
```

Comment:
I don't think `bt upgrade` needs to touch anything in `data/`, since it will be overwritten by `bt generate` anyway?

Reply:
not every Problem has a `generator.yaml`